components/ai_chat/resources/page/components/filter_menu/tabs_menu.test.tsx

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import './no_ce'
+
+import * as React from "react";
+import { AIChatContext, AIChatReactContext, defaultContext as defaultAIChatContext } from "../../state/ai_chat_context";
+import { ConversationReactContext, ConversationContext, defaultContext as defaultConversationContext } from "../../state/conversation_context";
+import { render } from "@testing-library/react";
+import TabsMenu from "./tabs_menu";
+import { ContentType } from 'gen/brave/components/ai_chat/core/common/mojom/ai_chat.mojom.m.js'
+const MockContext = (props: React.PropsWithChildren<Partial<AIChatContext & ConversationContext>>) => {
+  return <AIChatReactContext.Provider value={{
+    ...defaultAIChatContext,
+    ...props,
+  }}>
+    <ConversationReactContext.Provider value={{
+      ...defaultConversationContext,
+      ...props,
+    }}>
+      {props.children}
+    </ConversationReactContext.Provider>
+  </AIChatReactContext.Provider>
+}
+
+describe('TabsMenu', () => {
+  it('should render tabs', () => {
+    const { getByText } = render(<MockContext tabs={[{
+      contentId: 1,
+      title: 'Test 1',
+      url: {
+        url: 'https://tes1t.com',
+      },
+      id: 1
+    }, {
+      contentId: 2,
+      title: 'Test 2',
+      url: {
+        url: 'https://test2.com',
+      },
+      id: 2
+    }]}>
+      <TabsMenu />
+    </MockContext>)
+
+    expect(getByText('Test 1')).toBeInTheDocument()
+    expect(getByText('Test 2')).toBeInTheDocument()
+  })
```

Comment:
Any way to test that the `favicon` is displayed here?

Reply:
added a test!